src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -46,7 +47,7 @@ class SymfonyStyle extends OutputStyle
     public function __construct(InputInterface $input, OutputInterface $output)
     {
         $this->input = $input;
-        $this->bufferedOutput = new BufferedOutput($output->getVerbosity(), false, clone $output->getFormatter());
+        $this->bufferedOutput = new TrimmedBufferOutput($output->getVerbosity(), false, unserialize(serialize($output->getFormatter())), \DIRECTORY_SEPARATOR === '\\' ? 4 : 2);
```

Comment:
somebody knows how to perform a deep clone?

Issue is, Output and BufferOutput share the same formatter's styles, which has a state and count the number of closing tags. This does not works well when the same line is sent several time.

Note: previous implementation may suffer the same bug when the message contains less than 4 chars (ie. `</>X`)

Reply:
This specific bug is adressed by #39168